website/docs/r/cluster.html.markdown

```diff
@@ -17,7 +17,17 @@ description: |-
 ~> **IMPORTANT:**
 <br> &#8226; New Users: If you are not already using `mongodbatlas_cluster` for your deployment we recommend starting with the [`mongodbatlas_advanced_cluster`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_cluster).  `mongodbatlas_advanced_cluster` has all the same functionality as `mongodbatlas_cluster` but also supports multi-cloud clusters.  
 <br> &#8226; Free tier cluster creation (M0) is supported.
-<br> &#8226; Shared tier clusters (M0, M2, M5) cannot be upgraded to higher tiers via API or by this Provider.  WARNING! If you attempt to upgrade from an existing shared tier cluster that you manage with this Provider to a dedicated cluster (M10+) Terraform will see it as a request to destroy the shared tier cluster and as a request to create a dedicated tier cluster, i.e. Terraform will not see it as a request to upgrade.   If you accept the plan in this case the shared tier cluster would be destroyed and you would lose the data on that cluster. Do not attempt to upgrade from the shared to dedicated tier via this Provider, it is not supported!  
+<br> &#8226; Shared tier clusters (M0, M2, M5) are supported.
+```
+provider_instance_size_name = "M0"
+provider_name               = "TENANT"
+backing_provider_name       = "AWS"
+```
+<br> &#8226; Shared tier clusters can now be upgraded to higher tiers via API or by this Provider. WARNING! Any change from shared tier to a different instance size will be considered a tenant upgrade. When upgrading from shared tier, change the `provider_name` from "TENANT" and to your preferred provider, remove `backing_provider_name`, ex:
```

Comment:
Needs the hashes for the example config but couldn't get it to work in suggestion mode:
```suggestion
<br> &#8226; Shared tier clusters (M0, M2, M5) can be upgraded to dedicated tiers (M10+) via this provider. WARNING WHEN UPGRADING TENANT/SHARED CLUSTERS!!! Any change from shared tier to a different instance size will be considered a tenant upgrade. When upgrading from shared tier to dedicated simply change the `provider_name` from "TENANT"  to your preferred provider (AWS, GCP, AZURE) and remove the variable `backing_provider_name`, for example if you have an existing tenant/shared cluster and want to upgrade you Terraform config should be changed from:
```provider_instance_size_name = "M0"
```provider_name               = "TENANT"
```backing_provider_name       = "AWS"
To:
```